src/test/java/io/vertx/core/FutureTest.java

```diff
@@ -1627,4 +1628,91 @@ private void testListenersReportFailureOnContextAfterCompletion(Function<Context
     });
     waitUntil(() -> caught.size() == 1 && caught.get(0) == failure);
   }
+
+  @Test
+  @Repeat(times = 50)
```

Comment:
no need to repeat this test as there is no context involved so the current thread is always used